### ISSUE TITLE
Question: How can I copy a group from one file to another?

### ISSUE THREAD

**Issue by OWissett** [aldanor/hdf5-rust#278]
This is possible and easy to do using h5py, however, for me python is too slow so I am trying to use rust.

I seem to just be getting stuck with the issue that the types are not known at compile time. Is there are clean way of doing this or do I need to implement a copy function for each data type? which is very annoying it that is the case. Might be missing something, but it very hard to follow as the documentation is essentially non-existant. :( 

I want to essentially just have a list of keys which get copied from one file to another. It really feels like it should not be hard to do.

**Comment by mulimoen** [comment]
We do not yet wrap `H5Ocopy` which would allow this functionality. PRs are welcome!
